bool turn_off
---
bool auto_off
